scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -523,148 +523,190 @@ global.base_stats = { //tempory stats subject to change by anyone that wishes to
 			base_group : "astartes",
 			traits:["ancient","slow_and_purposeful","lead_example","zealous_faith",choose("still_standing","beast_slayer","lone_survivor")]
 	},			
+	/* TODO - check and tweak if necessary
+	"chapter_servitor":{
+			title : "Chapter Servitor",
+			strength:[36,4],
+			constitution:[36,3],
+			weapon_skill : [30,2,"max"],
+			ballistic_skill : [30,2,"max"],
+			dexterity:[36,3],
+			intelligence:0,
+			wisdom:0,
+			charisma :[10,1],
+			religion : "imperial_cult",
+			piety : [28,3],
+			luck : 10,
+			technology :[30,3],
+			skills: {weapons:{"Bolter":1, "Chainsword":1, "Combat Knife":1, "Bolt Pistol":1}},
+			start_gear:{"armour":"power_armour", "wep1":"Combat Knife", "gear":"Servo-arm"}, TODO - tweak and check if correct
+			base_group : "astartes",
+			traits:["lobotomized"],
+	},
+	*/
+	// TODO - add more hirelings on the imperial side...
 	"skitarii":{
 			title : "Skitarii",
-			strength:20,
-			constitution:28,
-			weapon_skill : [20,5],
-			ballistic_skill : [20,5],			
-			dexterity:25,
-			intelligence:25,
-			wisdom:10,
-			charisma :5,
+			strength : [12,1], // I don't think skitarii are particularly strong
+			constitution : [15,1],
+			weapon_skill : [12,1],
+			ballistic_skill : [20,1],			
+			dexterity : [25,1],
+			intelligence : [25,1],
+			wisdom : [10,1], // Hm, no, very unwise...
+			charisma : [5,1], // Talking in binary does not help to make many friends
 			religion : "cult_mechanicus",
-			piety : 20,
-			technology :30,
-			luck :5,
-			skills: {weapons:{"hellgun":1,}},	
-			start_gear:{"armour":"skitarii_armour", "wep1":"hellgun"},
+			piety : [20,1],
+			luck : 10, // I don't see the point to make them less lucky than space marines
+			technology : [30,1],
+			skills: {weapons:{"Hellgun":1,}},
+			start_gear:{"armour":"Skitarii Armour", "wep1":"Hellgun"},
 			base_group : "skitarii",
 	},
 	"tech_priest":{
-			strength:[12,1],
-			constitution:[30,1],
-			dexterity:[20,1],
-			weapon_skill : [15,5],
-			ballistic_skill : [15,5],				
-			intelligence:[30,3],
-			wisdom:[20,3],
-			charisma :[8,1],
-			religion : "cult_mechanicus",
 			title : "Tech Priest",
+			strength : [15,1],
+			constitution : [30,1],
+			weapon_skill : [15,1],
+			ballistic_skill : [25,3],				
+			dexterity : [25,3],
+			intelligence : [30,3],
+			wisdom : [20,2],
+			charisma : [30,2], // Considering their voicelines in DoW:DC and SS, I'd say they can have charisma
+			religion : "cult_mechanicus",
 			piety : [45,3],
-			luck :6,
-			technology :[55,3],
-			skills: {weapons:{"power_weapon":2,}},	
-			start_gear:{"armour":"dragon_scales", "wep1":"power_weapon"},
+			luck : 10,
+			technology : [55,3],
+			skills: {weapons:{"power_weapon":2,}}, // Does this even work?
+			start_gear:{"armour":"dragon_scales", "wep1":"power_weapon"}, // Gear items are not mentioned here
```

Comment:
You may want to fix names here, like you did with Skitarii.

Reply:
Ok...

Reply:
I hope "mobi" actually works...